test/selenium/pages/MonthlyContribution.scala

```diff
@@ -0,0 +1,55 @@
+package selenium.pages
+
+import org.scalatest.selenium.Page
+import selenium.util.{Browser, Config}
+
+object MonthlyContribution extends Page with Browser {
+
+  val url = s"${Config.supportFrontendUrl}/monthly-contributions"
+
+  private val stripeButton = id("qa-pay-with-card")
```

Comment:
Is it possible to pass a CSS selector to selenium? We are using BEM naming in CSS so it is possible to identify a button using CSS selector.